src/main/java/com/cloudogu/scm/review/pullrequest/api/PullRequestResource.java

```diff
@@ -58,13 +59,80 @@ public CommentRootResource comments() {
 
   @GET
   @Path("")
-  @Produces(MediaType.APPLICATION_JSON)
+  @Produces(PullRequestMediaType.PULL_REQUEST)
   public Response get(@Context UriInfo uriInfo, @PathParam("namespace") String namespace, @PathParam("name") String name, @PathParam("pullRequestId") String pullRequestId) {
     Repository repository = service.getRepository(namespace, name);
     PermissionCheck.checkRead(repository);
     return Response.ok(mapper.using(uriInfo).map(service.get(namespace, name, pullRequestId), repository)).build();
   }
 
+  @GET
+  @Path("approval")
+  @StatusCodes({
+    @ResponseCode(code = 200, condition = "success"),
+    @ResponseCode(code = 400, condition = "Invalid body"),
+    @ResponseCode(code = 401, condition = "not authenticated / invalid credentials"),
+    @ResponseCode(code = 403, condition = "not authorized, the current user does not have the privilege to update"),
+    @ResponseCode(code = 404, condition = "not found, no pull request with the specified id is available"),
+    @ResponseCode(code = 500, condition = "internal server error")
+  })
+  @Produces(PullRequestMediaType.PULL_REQUEST)
+  public Response getApproval(@Context UriInfo uriInfo, @PathParam("namespace") String namespace, @PathParam("name") String name, @PathParam("pullRequestId") String pullRequestId) {
+    Repository repository = service.getRepository(namespace, name);
+    PermissionCheck.checkRead(repository);
```

Comment:
I would prefer to check permission in service. Generally speaking you should move most logic from rest resource to services or handlers.